lib/src/lib.nr

```diff
@@ -541,6 +549,31 @@ fn left_byte_shift<N>(input: [u8; N], n: u64) -> [u8; N]
     out
 }
 
+#[test]
+fn memcpy_test()
+{
+    let mut dest = [0; 10];
+    let src = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20];
```

Comment:
Let's pick smaller arrays for tests. Arrays of size 2 and 3 are more than enough here and taking size 10 doesn't add to test robustness, but makes it harder to eyeball. Also - spaces after commas

Reply:
Ahmad doesn't use spaces after commas